docs/content/latest/deploy/checklist.md

```diff
@@ -139,6 +139,18 @@ In practice, the clock drift would have to be orders of magnitude higher in orde
 
 {{< /note >}}
 
+## DNS caching
+
+If you are using name servers instead of static ips you may end up doing aggressive DNS lookups.
```

Comment:
```suggestion
If you are using name servers instead of static IP addresses, you may end up with aggressive DNS lookups.
```

Reply:
Proving that I'm not as young as you, Paul Mockatpetris, a friend of mine and former coworker at Software.com, invented DNS (https://internethalloffame.org/inductees/paul-mockapetris). Of course, that's no reason to believe that I know what I'm talking about — I know enough to be dangerous. ;-)